cbt_export_changes_remote.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python
+
+import sys
+import XenAPI
+from python2_nbd_client import new_nbd_client
+import base64
+from bitstring import BitStream
+from xmlrpclib import ServerProxy
+
+BLOCK_SIZE = 64 * 1024
+
+def get_changed_blocks(host, export_name, bitmap):
+
+    bitmap = BitStream(bytes=base64.b64decode(bitmap))
+    with open('./bitmap', 'ab') as bit_out:
+        for bit in bitmap:
+            # The bits are written in the form "0" and "1" to the file
+            bit_out.write(str(int(bit)))

```

Comment:
Would be slightly cleaner to just look for 0 or 1, so that ppl know what to expect here.